### PR TITLE
fix(123done): configure stage paymentURL

### PR DESCRIPTION
**Pull request by jrgm**
r? - @lmorchard 

That product code came from Ben.

### CODE REVIEW

**Comment by jrgm** [comment]
Hrm. Did prettify make those other changes?

File: packages/123done/static/js/123done.js [hunk 1]
```diff
@@ -14,16 +14,18 @@ $(document).ready(function() {
...
   switch (window.location.host) {
     case '123done-latest.dev.lcip.org':
-      paymentURL = 'https://latest.dev.lcip.org/subscriptions/products/prod_Ex9Z1q5yVydhyk';
+      paymentURL =
+        'https://latest.dev.lcip.org/subscriptions/products/prod_Ex9Z1q5yVydhyk';
       break;
     case '123done-stage.dev.lcip.org':
-      paymentURL = 'TBD';
+      paymentURL =
+        'https://accounts.stage.mozaws.net/subscriptions/products/prod_FUUNYnlDso7FeB';
```
Comment (lmorchard) [inline packages/123done/static/js/123done.js]: This URL looks like the right pattern, though the product ID leads to a "no such product" error. Guessing we still need to switch to stage subhub & stripe where this product ID is available

**Comment by jrgm** [comment]
> assuming we still have config to tweak for subhub & stripe

Yes, but that **should** be the final url.

**Comment by lmorchard** [comment]
> Hrm. Did prettify make those other changes?

Also, yeah, probably prettify. Good changes to make though

**Pull request merged**